src/platform/silabs/wifi/SiWx/WifiInterface.cpp

```diff
@@ -661,53 +630,17 @@ sl_status_t bg_scan_callback_handler(sl_wifi_event_t event, sl_wifi_scan_result_
 
 /// NotifyConnectivity
 /// @brief Notify the application about the connectivity status if it has not been notified yet.
-///        Helper function for HandleDHCPPolling.
 void NotifyConnectivity(void)
 {
     VerifyOrReturn(!hasNotifiedWifiConnectivity);
+#if (CHIP_DEVICE_CONFIG_ENABLE_IPV4)
```

Comment:
according to existing logic, we used to trigger these based on the relevant IPv4 and IPv6 events, this gives me the impression that no matter what both IPv4 IPv6 has been acquired, is that understanding correct?
If yes, then do we need to keep track of the `hasNotifiedWifiConnectivity` event at all?